package.json

```diff
@@ -28,50 +28,48 @@
     "commander": "^2.19.0",
     "conjure-api": "^4.14.1",
     "conjure-client": "^2.4.1",
-    "fs-extra": "^5.0.0",
+    "fs-extra": "^10.1.0",
```

Comment:
This was done in https://github.com/palantir/conjure-typescript/pull/218/files already, no? Can you merge develop to make the diff smaller?

Reply:
Ah the base changed when that #218 merged, have rebased